makefile

```diff
@@ -26,16 +26,16 @@ remove-image:
 run:
 	$(COMMAND_RUN) --detach=true ${IMG_NAME} /bin/bash -c "bash" && \
 	$(COMMAND_COPY) && \
-	docker exec -it ${IMG_NAME} python restapi.py && \
+	docker exec -it ${IMG_NAME} . miniconda/bin/activate && python restapi.py && \
```

Comment:
I think you're right.  I don't think I actually tested this target, using instead ``docker-compose up``.  It might be worthwhile to remove the ``makefile``, as we've indicated deprecation in the past.

Reply:
I think there has been enough grace period ever since announced as deprecated (over a year since `BOPTEST v0.2.0` was released). I'd agree to remove it.